apps/meteor/app/api/server/v1/users.ts

```diff
@@ -555,6 +559,131 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'users.list/:status',
```

Comment:
Hey @MarcosSpessatto , was thinking on renaming this one to `users.listByStatus` to follow the other conventions on the users file, and passing the status as a query param. 

Wdyt?

Reply:
@tapiarafael Could we get some input on the necessity of the status as a url param here please?

Reply:
I don't see any problem in changing the endpoint to follow the convention. The front-end will just need to send the status via query param instead of sending it as URL param. @rique223